### PR TITLE
AzureCLI, PackerBuild, AzureVmssDeployment and ServiceFabricComposeDeploy loc handback

### PR DESCRIPTION
**Pull request by kuvinodms**

### CODE REVIEW

File: Tasks/PackerBuild/Strings/resources.resjson/zh-CN/resources.resjson [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
+<<<<<<< HEAD
```
Comment (sachinma) [inline Tasks/PackerBuild/Strings/resources.resjson/zh-CN/resources.resjson]: This is a merge issue

**Comment by sachinma** [comment]
Where is VmssDeployment? Would you create a separate PR

File: Tasks/PackerBuild/Strings/resources.resjson/zh-CN/resources.resjson (hunk 1 above)
Comment (bishal-pdMSFT) [inline Tasks/PackerBuild/Strings/resources.resjson/zh-CN/resources.resjson]: this is a conflict which needs to be resolved. Also, if this is the same in releases/m122 then that also needs to be fixed at priority

File: Tasks/PackerBuild/Strings/resources.resjson/zh-CN/resources.resjson [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
   "loc.friendlyName": "Build Machine Image (PREVIEW)",
+=======
+  "loc.friendlyName": "生成计算机映像",
+>>>>>>> 4fff3af9e... PackerBuild loc handback
   "loc.helpMarkDown": "[详细信息](https://go.microsoft.com/fwlink/?linkid=845329)",
   "loc.description": "Build machine image using Packer. This image can be used for Azure Virtual machine scale set deployment",
```
Comment (bishal-pdMSFT) [inline Tasks/PackerBuild/Strings/resources.resjson/zh-CN/resources.resjson]: Why is this not localized?

**Pull request merged**